packages/dendron-plugin-views/src/components/GraphFilterView.tsx

```diff
@@ -233,4 +273,59 @@ const FilterViewSection = ({
   );
 };
 
+const RadioButton = ({
+  value,
+  customCSS,
+}: {
+  value: GraphThemeEnum;
+  customCSS?: string;
+}) => {
+  let singleSelectOptions = Object.keys(GraphThemeEnum).map(
+    (k) => GraphThemeEnum[k as GraphThemeEnum]
+  );
+  if (!customCSS) {
+    singleSelectOptions = singleSelectOptions.filter(
+      (option) => option !== GraphThemeEnum.Custom
+    );
+  }
+  const ideDispatch = ideHooks.useIDEAppDispatch();
+  return (
+    <Radio.Group
+      onChange={(e) => {
+        updateGraphTheme(e.target.value);
+        ideDispatch(ideSlice.actions.setGraphTheme(e.target.value));
+      }}
+      value={value}
+    >
+      <Space direction="vertical">
+        {singleSelectOptions.map((option) => (
+          <Radio key={option} value={option}>
+            {option}
+          </Radio>
+        ))}
+      </Space>
+    </Radio.Group>
+  );
+};
+
+/**
+ * vscode message to update graphTheme selected by User.
+ * When the graph panel is disposed, this value is written back to Metadata Service.
```

Comment:
nit: "When the graph panel is disposed" - is this accurate? Do we not write back every time the radio button is changed?

Reply:
Each time the radio button is changed, we store the value in [this.defaultGraphTheme](https://github.com/dendronhq/dendron/pull/2806/files#diff-c05ef452a749e19d2b71f255ebbd218e4bca7d34abbbfc6aa587e50e266f973cR161) variable, and only write back to the meta file when the graph is disposed with the latest value.